apis/aisa.md

```diff
@@ -113,6 +113,7 @@ supported_languages:
   base_code: zh
   name: Chinese
   variant_name: Traditional Chinese
+more_languages_by_request: false
```

Comment:
I would remove this, the whole point of having a default in generate.py is that you don't need to include it in the data files.

Reply:
I think this was autogenerated when I ran generate.py

Reply:
I just added the `more_languages_by_request` boolean when TRUE in `apis.yml`, the rest was autogenerated.

Reply:
Ah, makes sense, pardon me, jetlag